.devcontainer/devcontainer.json

```diff
@@ -6,6 +6,8 @@
     "target": "dev-env",
   },
   "workspaceMount": "source=${localWorkspaceFolder},target=/code,type=bind,consistency=default",
+  // Uncomment the line below and replace src with the path to your local kitchen directory
+  // "mounts": ["type=bind,src=[path/to/kitchen],dst=/kitchen,readonly"],
```

Comment:
I tried really hard to load the path/to/kitchen from somewhere else so we could leave this line uncommented but couldn't find a way.  

Does the mount need to be readonly?  (might you want to modify kitchen from within your recipe window?)